pypeit/images/combineimage.py

```diff
@@ -273,8 +273,9 @@ def run(self, bias=None, flatimages=None, ignore_saturation=False, sigma_clip=Tr
                                                sigrej=sigrej, maxiters=maxiters)
             comb_img, comb_scl = img_list_out
             comb_rn2, comb_basev = var_list_out
-            comb_rn2[gpm] /= comb_scl[gpm]**2
-            comb_basev[gpm] /= comb_scl[gpm]**2
+            # Divide by the number of images that contributed to each pixel
```

Comment:
It may make more sense to change the variable name nstack to nframes to be more clear and consistent with the documentation.

Reply:
Done